### PR TITLE
chore: revert globals changes

### PR DESCRIPTION
**Pull request by spmonahan**
A follow up to #30967 (forgot to commit some PR feedback changes).

### CODE REVIEW

File: packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
-export const scrollIntoView = (target: HTMLElement | null | undefined, winArg?: Window | null) => {
+export const scrollIntoView = (target: HTMLElement | null | undefined) => {
```
Comment (spmonahan) [inline packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts:1]: Removing this isn't a problem as this is not part of the package's public API surface.

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Avatar-mount_base.html">655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Avatar-mount.html">652</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Button-mount_base.html">297</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Button-mount.html">306</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Field-mount_base.html">1159</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Field-mount.html">1140</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProvider-mount_base.html">717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProvider-mount.html">726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProviderWithTheme-mount_base.html">82</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProviderWithTheme-mount.html">89</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">40</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">32</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">86</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/MakeStyles-mount_base.html">882</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/MakeStyles-mount.html">869</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Persona-mount_base.html">1752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/Persona-mount.html">1752</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/SpinButton-mount_base.html">1449</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/SpinButton-mount.html">1405</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/SwatchPicker-mount_base.html">1595</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31470/perf-test-react-components/SwatchPicker-mount.html">1582</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr> | `100.456 kB`<br />`33.163 kB` | `100.458 kB`<br />`33.153 kB` | `2 B` <br />`-10 B` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr> | `101.089 kB`<br />`33.112 kB` | `101.091 kB`<br />`33.094 kB` | `2 B` <br />`-18 B` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ReactComponents.fixture.js'>react-components: entire library</abbr> | `1.092 MB`<br />`269.741 kB` | `1.092 MB`<br />`269.718 kB` | `1 B` <br />`-23 B` |
| <samp>react-tag-picker</samp> <br /> <abbr title='bundle-size/Package.fixture.js'>@fluentui/react-tag-picker - package</abbr> | `181.024 kB`<br />`54.556 kB` | `181.025 kB`<br />`54.547 kB` | `1 B` <br />`-9 B` |
| <samp>react-timepicker-compat</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr> | `103.248 kB`<br />`34.54 kB` | `103.256 kB`<br />`34.524 kB` | `8 B` <br />`-16 B` |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> | `100.003 kB`<br />`30.524 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr> | `78.817 kB`<br />`22.543 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr> | `49.299 kB`<br />`15.837 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr> | `20.107 kB`<br />`7.973 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr> | `63.316 kB`<br />`20.017 kB` |
| <samp>react-breadcrumb</samp> <br /> <abbr title='bundle-size/index.fixture.js'>@fluentui/react-breadcrumb - package</abbr> | `114.486 kB`<br />`31.821 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr> | `37.103 kB`<br />`10.784 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr> | `43.514 kB`<br />`12.074 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr> | `41.884 kB`<br />`12.132 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr> | `49.896 kB`<br />`13.724 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr> | `53.03 kB`<br />`12.543 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr> | `101.184 kB`<br />`29.049 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr> | `94.218 kB`<br />`27.333 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr> | `14.354 kB`<br />`5.795 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr> | `16.618 kB`<br />`6.555 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr> | `14.418 kB`<br />`5.931 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `69.141 kB`<br />`20.157 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `212.631 kB`<br />`61.228 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `44.442 kB`<br />`14.607 kB` |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr> | `220.038 kB`<br />`62.373 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr> | `98.817 kB`<br />`29.825 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr> | `152.023 kB`<br />`45.98 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu  (including selectable components)</abbr> | `154.709 kB`<br />`46.477 kB` |
| <samp>react-message-bar</samp> <br /> <abbr title='bundle-size/Default.fixture.js'>MessageBar (all components)</abbr> | `24.411 kB`<br />`9.114 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr> | `56.19 kB`<br />`17.726 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr> | `127.766 kB`<br />`40.096 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `8.39 kB`<br />`2.64 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr> | `162.843 kB`<br />`46.317 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr> | `42.48 kB`<br />`13.866 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr> | `131.934 kB`<br />`36.682 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr> | `70.334 kB`<br />`20.026 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr> | `68.977 kB`<br />`19.63 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr> | `15.165 kB`<br />`6.134 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr> | `28.994 kB`<br />`9.518 kB` |
| <samp>react-tags</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr> | `82.911 kB`<br />`24.642 kB` |
| <samp>react-toast</samp> <br /> <abbr title='bundle-size/Toast.fixture.js'>Toast (including Toaster)</abbr> | `99.625 kB`<br />`30.033 kB` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a9defa95f62960e1584dd9d6d07db57d1fdeec4b'>a9defa95f62960e1584dd9d6d07db57d1fdeec4b</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts (hunk 1 above)
Comment (fabricteam) [inline packages/react-components/react-aria/src/activedescendant/scrollIntoView.ts:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31470&commitId=ab70d37b312e60327cc7fdc4bde373c8e7117526&env=prod&clientType=FLUENTUI)


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by layershifter — APPROVED** [review]

**Pull request merged**